pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -4634,7 +4634,7 @@ protected void internalGetLastMessageId(AsyncResponse asyncResponse, boolean aut
                     });
                 }).exceptionally(ex -> {
                     // If the exception is not redirect exception we need to log it.
-                    if (!isRedirectException(ex)) {
+                    if (!isNot307And404Exception(ex)) {
```

Comment:
it seems that this line only log the 307 and 404 exceptions. not(not 307 or not 404)

Reply:
Good pointer! Now the implementation is only not `307 | 404`(so far, seems broker will not throw other `30x`)